### PR TITLE
Fix: Dependabot pipeline

### PR DESCRIPTION
**Pull request by laisspportugal**
**User-Facing Changes**

**Description**
In order to fix the dependabot pipeline we are changing the node version. Also a small change was added to fix the README description.

### CODE REVIEW

File: .github/workflows/dependabot-fix.yml [hunk 1]
```diff
@@ -15,7 +15,8 @@ on:
 jobs:
   fix:
     runs-on: ubuntu-latest
-    if: github.actor == 'dependabot[bot]'
+    # Remove to testing. Put it again as soon as possible
```
Comment (laisspportugal) [inline .github/workflows/dependabot-fix.yml:18]: please undo this change

Comment (aneuwald-ctw) [reply .github/workflows/dependabot-fix.yml]: done

**Review by aneuwald-ctw — APPROVED** [review]

**Review by aneuwald-ctw — APPROVED** [review]
